packages/template-retail-react-app/app/components/toggle-card/index.jsx

```diff
@@ -49,7 +49,7 @@ export const ToggleCard = ({
                         <Heading
                             fontSize="lg"
                             lineHeight="30px"
-                            color={disabled && !editing && 'gray.400'}
+                            color={disabled && !editing && 'gray.600'}
```

Comment:
This is reported from axe tool, so I bumped the color up to be a bit darker for color contrast requirement